go/cmd/dolt/commands/engine/jwtplugin.go

```diff
@@ -23,15 +23,11 @@ import (
 	"github.com/dolthub/go-mysql-server/sql/mysql_db"
 	"github.com/sirupsen/logrus"
 
+	"github.com/dolthub/dolt/go/libraries/doltcore/servercfg"
 	"github.com/dolthub/dolt/go/libraries/utils/jwtauth"
 )
 
-type JwksConfig struct {
-	Name        string            `yaml:"name"`
-	LocationUrl string            `yaml:"location_url"`
-	Claims      map[string]string `yaml:"claims"`
-	FieldsToLog []string          `yaml:"fields_to_log"`
-}
+type JwksConfig servercfg.ServerJwksConfig
```

Comment:
Any reason to keep this type alias?

Reply:
Good call.  I was thinking that JwksConfig had some methods attached to it in this package... but it's totally unneeded.